ios/FluentUI/Avatar/Avatar.swift

```diff
@@ -73,7 +73,10 @@ import SwiftUI
 }
 
 /// View that represents the avatar.
-public struct Avatar: View, ConfigurableTokenizedControl {
+public struct Avatar: View, TokenizedControlView {
+    public typealias TokenSetKeyType = AvatarTokenSet.Tokens
```

Comment:
Does this also need `public typealias TokenSetType = AvatarTokensSet`?

Reply:
Apparently not--I'll remove that from the other controls as well.